server/datastore/mysql/hosts.go

```diff
@@ -4155,8 +4155,7 @@ func (ds *Datastore) AggregatedMDMSolutions(ctx context.Context, teamID *uint, p
 
 func (ds *Datastore) GenerateAggregatedMunkiAndMDM(ctx context.Context) error {
 	var (
-		// TODO(android): add android to this list?
-		platforms = []string{"", "darwin", "windows", "ios", "ipados"}
+		platforms = []string{"", "darwin", "windows", "ios", "ipados", "android"}
```

Comment:
This is just adding `android` to the list of platforms we generate MDM stats for, in preparation for the rest of the work for this ticket. I don't think the rest can be done just yet (I may be wrong?), we just need to create/update the `host_mdm` entry for Android hosts ("when" is the main question - immediately as "enrolled" when the host is created? and when do we update it as "unenrolled" if it turns off MDM?). We also need to ensure that the MDM solution will have Fleet's server URL as URL in `mobile_device_management_solutions` when creating the entry in `host_mdm`.